tests/Architecture/ConditionableTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Illuminate\Support\Traits\Conditionable;
+use Native\Desktop;
+
+use function Pest\version;
+
+describe('architecture', function () {
+
+    arch('ensure api is conditionable')
+        ->expect([
+            Desktop\App::class,
+            Desktop\Dock::class,
+            Desktop\Alert::class,
+            Desktop\Clipboard::class,
+            Desktop\AutoUpdater::class,
+            Desktop\ChildProcess::class,
+            Desktop\Notification::class,
+            Desktop\Menu\Menu::class,
+            Desktop\Windows\Window::class,
+            Desktop\Windows\WindowManager::class,
+
+            Desktop\Fakes\ChildProcessFake::class,
+            Desktop\Fakes\WindowManagerFake::class,
+        ])->each->toUseTrait(Conditionable::class);
+
+})->skip(function () {
+    // Only run test when pest version is at least 3
+    return version_compare(version(), '3.0.0', '<');
```

Comment:
The test was failing on `prefer-lowest` when using pest 2. 

I admit that this workaround is a bit janky. But if this runs on the higher versions i think that is enough.